tools/test/TestCSVFileImport.cc

```diff
@@ -53,3 +53,27 @@ TEST (TestCSVFileImport, test10rows) {
   EXPECT_EQ(expected, output);
   EXPECT_EQ("", error);
 }
+
+TEST (TestCSVFileImport, testTimezoneOption) {
+  // create an ORC file from importing the CSV file
+  const std::string pgm1 = findProgram("tools/src/csv-import");
+  const std::string csvFile =
+    findExample("TestCSVFileImport.testTimezoneOption.csv");
+  const std::string orcFile = "/tmp/test_csv_import_test_timezone_option.orc";
+  const std::string schema = "'struct<_a:timestamp>'";
+  std::string output;
+  std::string error;
+
+  std::string option = "--timezone=America/Los_Angeles";
```

Comment:
Maybe, this test case always succeeds at `America/Los_Angeles` timezone? If then, can we choose less popular timezone?

Reply:
If we don't set timezone for the WriterOptions, the writer will write the timestamp with the GMT timezone. And the orc-contents tool will scan the content in the GMT timezone.This case should not be successful?

Reply:
We can add another test to use timezone as Europe/Paris or Asia/Shanghai.

Reply:
@coderex2522 . Yes, the test case should fail without your patch and should pass with your patch. That's the purpose of verification of your code's contribution.

And, +1 for @wgtmac .

Reply:
> We can add another test to use timezone as Europe/Paris or Asia/Shanghai.

Add the timezone 'Europe/Paris' test case done.